app/i18n/es/people.ts

```diff
@@ -212,12 +232,14 @@ const people: People = {
           'Professor of Electrical and Computer Engineering - University of Washington',
           'Ph.D. Electrical Engineering, Stanford',
         ],
+        link: 'https://people.ece.uw.edu/atlas/',
       },
       {
         img: NobuyoOzawa,
         name: 'Nobuyo Ozawa',
```

Comment:
Based on Figma and https://virufy.org/en/our-org, I think `Nobuyo` should be replaced with `Nobuyoshi`. Double check the other `/people.ts` files as well. If you can also rename the image file to reflect the correct spelling, it would be appreciated (ie. `NobuyoOzawa` to `NobuyoshiOzawa`).

Reply:
done